mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test.go

```diff
@@ -100,4 +99,27 @@ resource "google_scc_v2_folder_mute_config" "default" {
   parent               = "folders/%{folder_id}"
 }
 `, context)
-}
\ No newline at end of file
+}
+func testAccCheckSecurityCenterV2FolderMuteConfigDestroyProducer(t *testing.T) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		config := acctest.Provider.Meta().(*acctest.Config)
```

Comment:
This will not compile. There is no `Config` type in `acctest`.

Reply:
what we can use instead of that.